lib/_internal/file_picker_web.dart

```diff
@@ -185,6 +186,45 @@ class FilePickerWeb extends FilePicker {
     return files == null ? null : FilePickerResult(files);
   }
 
+  @override
+  Future<String?> saveFile({
+    String? dialogTitle,
+    String? fileName,
+    String? initialDirectory,
+    FileType type = FileType.any,
+    List<String>? allowedExtensions,
+    Uint8List? bytes,
+    bool lockParentWindow = false,
+  }) async {
+    if (bytes == null) {
+      throw ArgumentError(
+          'The "bytes" parameter is required for saveFile on web');
+    }
+
+    if (fileName == null || p.extension(fileName).isEmpty) {
+      throw ArgumentError(
```

Comment:
The indent is a bit off here. I would also split the two checks, so that the error message can be improved.

```suggestion
   if (fileName == null || fileName.isEmpty) {
      throw ArgumentError(
        'A file name is required when saving a file on the web.'
      );   
   }
   
   if (p.extension(fileName).isEmpty) {
     throw ArgumentError('The file name should include a valid file extension.');
   }
```